modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -233,6 +231,58 @@ function FieldBase({
 			type === 'radio');
 	const showPopover = fieldName === 'inputMaskFormat';
 
+	useLayoutEffect(() => {
+		console.log(type);
+		focusRequiredElement();
+	});
+
+	function focusRequiredElement() {
+		let errorInput = null;
+
+		const errorContainer = document.querySelector('.form-feedback-group');
+
+		if (errorContainer) {
+			switch (type) {
```

Comment:
@joaovictoralvesmartins, where you able to find out why this fix is not needed for the fields Text and Numeric? Maybe there is a code performing a similar action (focus) and would be nice we move it to this new code to make easier the future maintenance. :wink:

Reply:
@natocesarrego, unfortunately i couldn't find why this fix is not needed for Text and Numeric inputs, but i can search a bit more.